.github/workflows/ci.yml

```
@@ -0,0 +1,25 @@
+name: ci
+
+on:
+  push:
+    branches-ignore:
+      - "master"
+      - "develop"
+      - "release/**"
+  pull_request:
+
+jobs:
+  test:
+    name: test
+    strategy:
+      matrix:
+        os: [ ubuntu-20.04, macOs-latest, windows-latest ]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 14
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
```

As we are not limited by concurrency anymore, we should consider to add node LTS versions back to the build matrix again.

```suggestion
      matrix:
        os: [ ubuntu-20.04, macOs-latest, windows-latest ]
        node: [ 10, 12, 14 ]
    runs-on: ${{ matrix.os }}
    steps:
      - uses: actions/checkout@v2
      - name: Use Node.js {{ matrix.os }}
        uses: actions/setup-node@v1
        with:
          node-version: {{ matrix.os }}
```